src/js/tech/tech.js

```diff
@@ -218,12 +218,13 @@ class Tech extends Component {
       return;
     }
 
+    let player = this.player();
     let textTracksChanges = function() {
       let updateDisplay = Fn.bind(this, function() {
-        this.trigger('texttrackchange');
+        player.trigger('texttrackchange');
```

Comment:
Sounds good. The event architecture is getting a little opaque; I'll leave it to you core developers, but it'd be nice to have some docs on it at some point in the future ;-)